src/modules/transaction/types.ts

```diff
@@ -27,3 +30,7 @@ export enum NetworkName {
   kovan = 'kovan',
   rinkeby = 'rinkeby'
 }
+
+interface Receipt {
+  logs: any[]
```

Comment:
Is it possible to type this? or is it way too dynamic?

Reply:
It has events logs on it, `args` of events could be dynamic. 

I can do something like
```ts
logs: {
name: string,
args: any,
[key: string]: any
}
```

Is it worth?

Reply:
think so! it allows VSCode to autocomplete .name and .args at least